src/tokens/PositionIcon.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { Image, StyleSheet, View } from 'react-native'
+import FastImage from 'react-native-fast-image'
+import { useSelector } from 'src/redux/hooks'
+import { networksIconSelector } from 'src/tokens/selectors'
+import { Spacing } from 'src/styles/styles'
+import { Position } from 'src/positions/types'
+
+const networkImageSize = 12
+const networkImagePosition = 20
+
+export const PositionIcon = ({ position, testID }: { position: Position; testID?: string }) => {
```

Comment:
nit: easier to parse visually and less chars
```suggestion
export function PositionIcon({ position, testID }: { position: Position; testID?: string }) {
```